source/core/parameters.cc

```diff
@@ -871,6 +871,46 @@ namespace Parameters
     prm.leave_subsection();
   }
 
+  void
+  Laser_FreeSurfaceRadiation::declare_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("free surface radiation");
+    {
+      prm.declare_entry(
+        "enable",
+        "false",
+        Patterns::Bool(),
+        "Enable radiation at the free surface (air/metal interface) <true|false>");
+      prm.declare_entry("Stefan-Boltzmann constant",
+                        "0.000000056703",
```

Comment:
you can use exponential notation
I suggest writing it as 5.6703e-8 instead of putting tha gazillion zeroes. It's easier the my brain :).

Reply:
Done :)